src/olympia/versions/tests/test_commands.py

```diff
@@ -94,7 +94,7 @@ def test_full(self):
             [
                 ['addon_id'],
                 *[[str(addon.pk)] for addon in addons_to_modify],
-                *[[str(addon.pk)] for addon in addons_to_ignore_promoted],
+                *[[str(addon.pk) for addon in addons_to_ignore_promoted]],
```

Comment:
It's a straight revert from the sneak peek commit, where it was included as an intentional fix to this test. Ultimately that line is about adding data that is ignored so it doesn't fail the test either way, but it's more correct the other way, so I'll put it back

Reply:
With `*[[str(addon.pk) for addon in addons_to_ignore_promoted]],` the CSV contains
```
(...)
id1,id2
(...)
```
The command ignores it because the row is not an id.

With `*[[str(addon.pk)] for addon in addons_to_ignore_promoted],` (fix that was reverted as part of reverting the sneak peek commit) the CSV contains
```
(...)
id1
id2
(...)
```

The command ignores it because the rows are recommended/line add-on ids.

Reply:
https://github.com/mozilla/addons-server/pull/21590/commits/72157d74fd83daa0c1d45e8eea97571f9b9eb4f5